src/grokcore/component/compat.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import martian
+from zope.interface import classImplements
+from zope.interface.interfaces import IInterface
+
+if sys.version_info[0] < 3:
+    from zope.interface import implements
+else:
+
+    class implements(martian.Directive):
+        scope = martian.CLASS
+        store = martian.ONCE
+        default = None
+        
+        def factory(self, *interfaces):
+            for interface in interfaces:
+                if not IInterface.providedBy(interface):
+                    raise Exception('%s is not a interface' % repr(interface))
+            return interfaces
+
+
+    class ClassImplementerGrokker(martian.ClassGrokker):
```

Comment:
See the exception if i use them on python 3: ***TypeError: Class advice impossible in Python3.  Use the @implementer class decorator instead.***
The goal of this Grokker is to reimplement class advice same way as in python 2. In python 2 we can directly use the implementer directive without a grokker.